cmd/crl-updater/main.go

```diff
@@ -36,13 +37,30 @@ type Config struct {
 		// in CCADB MUST be updated.
 		NumShards int
 
+		// ShardWidth is the amount of time (width on a timeline) that a single
+		// shard should cover. Ideally, NumShards*ShardWidth should be an amount of
+		// time noticeably larger than the current longest certificate lifetime,
+		// but the updater will continue to work if this is not the case (albeit
+		// with more confusing mappings of serials to shards).
+		// WARNING: When this number is changed, revocation entries will move
+		// between shards.
+		ShardWidth cmd.ConfigDuration
+
+		// LookbackPeriod is how far back the updater should look for revoked expired
+		// certificates. We are required to include every revoked cert in at least
+		// one CRL, even if it is revoked seconds before it expires, so this must
+		// always be greater than the UpdatePeriod, and should be increased when
+		// recovering from an outage to ensure continuity of coverage.
+		LookbackPeriod cmd.ConfigDuration
+
 		// CertificateLifetime is the validity period (usually expressed in hours,
 		// like "2160h") of the longest-lived currently-unexpired certificate. For
 		// Let's Encrypt, this is usually ninety days. If the validity period of
 		// the issued certificates ever changes upwards, this value must be updated
 		// immediately; if the validity period of the issued certificates ever
 		// changes downwards, the value must not change until after all certificates with
 		// the old validity period have expired.
+		// DEPRECATED: This config value is no longer used.
```

Comment:
You should include a TODO to remove this.

```suggestion
		// DEPRECATED: This config value is no longer used.
```